.github/workflows/test.yml

```
@@ -5,8 +5,37 @@ on:
+
+      - uses: actions/setup-node@v4
+        env:
+          FORCE_COLOR: 0
+        with:
+          node-version: ${{ matrix.node }}
+          cache: yarn
+
+      - run: yarn --prefer-offline --frozen-lockfile
+      - run: yarn test
+
+      - uses: codecov/codecov-action@v3
```

actionlint (1.7.11)

[error] the runner of "codecov/codecov-action@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)